Taskfile.yml

```diff
@@ -0,0 +1,89 @@
+# https://taskfile.dev
+# To run on a system without task installed, `./scripts/run_task.sh` will execute it with `go run`.
+# If in the nix dev shell, `task` is available.
+
+version: '3'
+
+tasks:
+  default: ./scripts/run_task.sh --list
+
+  actionlint:
+    desc: Run the actionlint.sh
+    cmd: bash ./scripts/actionlint.sh # tests.yml
```

Comment:
AvalancheGo doesn't do any tests on Windows but I missed that - to ensure that tests continue to run correctly on Windows, the scripts have to ran in bash directly as Windows doesn't know how to execute the .sh e.g. `fork/exec D:\a\subnet-evm\subnet-evm\scripts\build.sh: %1 is not a valid Win32 application` 

I tried and failed a couple other more elegant solutions than just appending bash to no avail.

Reply:
I don't think we need to run actionlint with Windows, actually we can even start removing CI tests for Windows as it's not supported in avalanchego anymore.

Reply:
Done!